README.md

```diff
@@ -8,11 +8,8 @@ The landing page must be built in order to get its static parts compiled.
 
 ### Docker
 
-- Clone, Build docker image: `docker build -t tgstation .`
-- Or download it instead `docker pull ghcr.io/tgstation.org/tgstation-web:master`
```

Comment:
This url needs to be listed in the repo, as somebody building this for a k8s or swarm deployment (like me) can't use docker-compose to build and would need to generate a custom compose file that will need to use this url.

Reply:
I have no idea what that means...